flixel/addons/tile/FlxTilemapExt.hx

```diff
@@ -713,21 +890,21 @@ class FlxTilemapExt extends FlxTilemap
 	 */
 	private function setSlopeProperties():Void
 	{
-		for (i in 0..._slopeFloorLeft.length)
+		for (i in _slopeFloorLeft)
```

Comment:
I like the change to for each loops, but `i` doesn't really make sense anymore as a variable name in that case, since it generally stands for "index". Should be `tile` or `slope`.